### PR TITLE
fixes and adds additional parameters

### PR DESCRIPTION
**Pull request by birdage**
@DanielJMaher we will most likely want to talk about this, as i had to make a small change to the plotting page.

### CODE REVIEW

File: ooiui/templates/science/plot.html [hunk 1]
```diff
@@ -155,19 +137,22 @@
...
                                     </div>
                                 </div>
                             </div>
+                            <div class="panel-footer clearfix">
+                                <div class="pull-left">
+                                    <input id="showAdditionalParameters" type="checkbox" class="" name="Show Additional Parameters" value="showAdditionalParameters"> Show Additional Parameters<br>
+                                </div>
+                                <div class="pull-center">
+                                    <a href="#" class="btn btn-primary update-plot">Plot</a>
+                                    <small class="pull-right">  * Data is decimated. Maximum of 1000 points shown.</small>
```
Comment (DanielJMaher) [inline ooiui/templates/science/plot.html]: data ARE decimated

Comment (oceanzus) [inline ooiui/templates/science/plot.html]: Decimated data.

Comment (birdage) [inline ooiui/templates/science/plot.html]: can someone just give me the language to use "* Data are decimated. Maximum of 1000 points shown" is what i have right now

Comment (DanielJMaher) [inline ooiui/templates/science/plot.html]: "Data are decimated. Maximum of 1000 points shown."

**Comment by DanielJMaher** [comment]
@oceanzus Does this cause your PR to not merge nicely?

**Comment by birdage** [comment]
@DanielJMaher i moved it. merge if your happy with it

screen shot 2016-06-21 at 2 19 22 pm

**Comment by oceanzus** [comment]
@DanielJMaher Merge away.  I'll deal with it downstream.

**Pull request merged**